UI/data/themes/Rachni.qss

```diff
@@ -0,0 +1,1032 @@
+/***************************************************************************/
+/*   Copyright (C) 2017-2018 by Joel Bethke <joel.bethke@gmail.com>        */
+/*                                                                         */
+/*                                                                         */
+/*   This program is free software: you can redistribute it and/or modify  */
+/*   it under the terms of the GNU General Public License as published by  */
+/*   the Free Software Foundation, either version 2 of the License, or     */
+/*   (at your option) any later version.                                   */
```

Comment:
Ah, you're right.  I was thinking of [this commit](https://github.com/jp9000/obs-studio/commit/8298fa4dc7108f2e4323f26f04ab19fc6d43e376), but forgot it was GPLv2+ and not strictly GPLv2.  My mistake.